android/build.gradle

```diff
@@ -39,6 +39,7 @@ android {
         versionName "2.0.1"
 
         consumerProguardFiles 'consumer-rules.pro'
+        missingDimensionStrategy 'dependencyVersions', 'latestDependencies', 'unityIAP'
```

Comment:
Could we add a comment here specifying what it does? 

I figure that this will try to choose the latest dependencies first, but if there's an incompatibility with other requirements for the same app, it'll try the flavor `unityIAP`, is that correct?

Reply:
I am not sure what this is for either @beylmk . Can you add some context?

Reply:
@aboedo yes, that's correct. we have to specify the flavor of purchases-android that we want to depend on, and this defaults to latestDependencies. 

we could instead make purchases-hybrid-common flavors, which in turn pull a specific purchases-android flavor. i figured that could be a follow-up ticket

Reply:
see points 2/3 in the TLDR at this link: https://kiranrao.in/blog/2020/03/31/gradle-missing-flavors/

according to that, we probably do want to make PHC flavors

Reply:
I am actually working on adding the flavors so we can improve Unity by A LOT

Reply:
I think maybe this was required because you were doing local development and pointing to your copy of purchases-android 🤔